### DISCUSSION TITLE
unsupported type: 5: Spectral_Band_Replication

### DISCUSSION THREAD

**Discussion by Kiennh** [bluenviron/mediamtx#1067] [Questions] [answered]
When trying to push rtmp stream contains a HE-AAC audio channel to server, i got the error Unsupported type: [5 (SBR)](https://wiki.multimedia.cx/index.php/Spectral_Band_Replication)

So i look into the [gortsplib](https://github.com/aler9/gortsplib) library  i found that the library only support for [AAC-LC](https://github.com/aler9/gortsplib/blob/main/pkg/aac/audiotype.go) - with type 2.

Are there any reason for support only AAC-LC ? and how we can receive and HE-AAC audio stream? 

Look at the document, i can see that rtsp-simple-server can handle bot mp3 and aac audio stream, and I think that there're may be have a workaround solution for this (mp3 is type 5 too)
```
Each stream can have multiple video and audio tracks, encoded with any codec, including H264, H265, VP8, VP9, MPEG2, MP3, AAC, Opus, PCM, JPEG

```

```
2022/08/04 17:11:57 INF rtsp-simple-server v0.0.0
2022/08/04 17:11:57 DEB [path pull1] created
2022/08/04 17:11:57 DEB path manager created
2022/08/04 17:11:57 INF [RTSP] listener opened on :8554 (TCP), :8000 (UDP/RTP), :8001 (UDP/RTCP)
2022/08/04 17:11:57 INF [RTMP] listener opened on :19352
2022/08/04 17:11:57 INF [HLS] listener opened on :8888
2022/08/04 17:11:58 INF [RTMP] [conn [::1]:56567] opened
2022/08/04 17:11:58 INF [RTMP] [conn [::1]:56567] closed (unsupported type: 5)

```

**Comment by aler9** [comment, marked as answer]
Hello, we'd be glad to support HE-AAC too. I've created an issue to track this enhancement, please move the discussion there #1068
